agent/build.gradle

```diff
@@ -54,12 +54,12 @@ shadowJar {
 	// Needed as a workaround for https://github.com/johnrengelman/shadow/issues/521
 	inputs.property("debug", project.hasProperty("debug"))
 
-	archiveBaseName = 'teamscale-jacoco-agent'
+	baseName = 'teamscale-jacoco-agent'
```

Comment:
baseName is deprecated and will be removed in the next Gradle version. The documentation suggests to use `archiveBaseName` instead. Are we sure that this is needed to get it back working?